FAILED: Akka Typed: Performance Spec

Failed multiple times (both nightly and pr validation recently):

https://jenkins.akka.io:8498/job/akka-multi-node/1008/consoleFull
https://jenkins.akka.io:8498/job/akka-multi-node-nightly/2251/consoleFull
https://jenkins.akka.io:8498/job/akka-nightly/2124/consoleFull
```
[info]   - 09 when using 8 pairs with 10 messages *** FAILED *** (30 seconds, 6 milliseconds)
[info]     java.util.concurrent.TimeoutException: Futures timed out after [30 seconds]
[info]     at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:219)
[info]     at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:223)
[info]     at scala.concurrent.Await$$anonfun$result$1.apply(package.scala:116)
[info]     at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
[info]     at scala.concurrent.Await$.result(package.scala:116)
[info]     at akka.typed.TypedSpec.await(TypedSpec.scala:40)
[info]     at akka.typed.TypedSpec.sync(TypedSpec.scala:58)
[info]     at akka.typed.PerformanceSpec$A$u0020static$u0020behavior$must$u0020be$u0020fast$.09$u0020when$u0020using$u00208$u0020pairs$u0020with$u002010$u0020messages(PerformanceSpec.scala:67)
[info]     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[info]     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
[info]     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[info]     at java.lang.reflect.Method.invoke(Method.java:622)
[info]     at org.scalatest.SpecLike$$anonfun$register$1$1$$anonfun$2.apply(SpecLike.scala:124)
[info]     at org.scalatest.SpecLike$$anonfun$register$1$1$$anonfun$2.apply(SpecLike.scala:122)
[info]     at org.scalatest.Transformer$$anonfun$apply$1.apply(Transformer.scala:22)
[info]     at org.scalatest.Transformer$$anonfun$apply$1.apply(Transformer.scala:22)
[info]     at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]     at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]     at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]     at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]     at org.scalatest.SpecLike$$anon$1.apply(SpecLike.scala:259)
[info]     at org.scalatest.Suite$class.withFixture(Suite.scala:1121)
[info]     at org.scalatest.Spec.withFixture(Spec.scala:1397)
[info]     at org.scalatest.SpecLike$class.invokeWithFixture$1(SpecLike.scala:256)
[info]     at org.scalatest.SpecLike$$anonfun$runTest$1.apply(SpecLike.scala:268)
[info]     at org.scalatest.SpecLike$$anonfun$runTest$1.apply(SpecLike.scala:268)
[info]     at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
[info]     at org.scalatest.SpecLike$class.runTest(SpecLike.scala:268)
[info]     at org.scalatest.Spec.runTest(Spec.scala:1397)
[info]     at org.scalatest.SpecLike$$anonfun$runTests$1.apply(SpecLike.scala:314)
[info]     at org.scalatest.SpecLike$$anonfun$runTests$1.apply(SpecLike.scala:314)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:413)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]     at scala.collection.immutable.List.foreach(List.scala:381)
[info]     at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]     at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:390)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:427)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]     at scala.collection.immutable.List.foreach(List.scala:381)
[info]     at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]     at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:390)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:427)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]     at scala.collection.immutable.List.foreach(List.scala:381)
[info]     at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]     at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:396)
[info]     at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:483)
[info]     at org.scalatest.SpecLike$class.runTests(SpecLike.scala:314)
[info]     at org.scalatest.Spec.runTests(Spec.scala:1397)
[info]     at org.scalatest.Suite$class.run(Suite.scala:1423)
[info]     at org.scalatest.Spec.org$scalatest$SpecLike$$super$run(Spec.scala:1397)
[info]     at org.scalatest.SpecLike$$anonfun$run$1.apply(SpecLike.scala:319)
[info]     at org.scalatest.SpecLike$$anonfun$run$1.apply(SpecLike.scala:319)
[info]     at org.scalatest.SuperEngine.runImpl(Engine.scala:545)
[info]     at org.scalatest.SpecLike$class.run(SpecLike.scala:319)
[info]     at akka.typed.TypedSpec.org$scalatest$BeforeAndAfterAll$$super$run(TypedSpec.scala:25)
[info]     at org.scalatest.BeforeAndAfterAll$class.liftedTree1$1(BeforeAndAfterAll.scala:257)
[info]     at org.scalatest.BeforeAndAfterAll$class.run(BeforeAndAfterAll.scala:256)
[info]     at akka.typed.TypedSpec.run(TypedSpec.scala:25)
[info]     at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:444)
[info]     at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:651)
[info]     at sbt.TestRunner.runTest$1(TestFramework.scala:76)
[info]     at sbt.TestRunner.run(TestFramework.scala:85)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1$$anonfun$apply$8.apply(TestFramework.scala:202)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1$$anonfun$apply$8.apply(TestFramework.scala:202)
[info]     at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:185)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1.apply(TestFramework.scala:202)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1.apply(TestFramework.scala:202)
[info]     at sbt.TestFunction.apply(TestFramework.scala:207)
[info]     at sbt.Tests$.sbt$Tests$$processRunnable$1(Tests.scala:239)
[info]     at sbt.Tests$$anonfun$makeSerial$1.apply(Tests.scala:245)
[info]     at sbt.Tests$$anonfun$makeSerial$1.apply(Tests.scala:245)
[info]     at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
[info]     at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
[info]     at sbt.std.Transform$$anon$4.work(System.scala:63)
[info]     at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
[info]     at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
[info]     at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[info]     at sbt.Execute.work(Execute.scala:235)
[info]     at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
[info]     at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
[info]     at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
[info]     at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
[info]     at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
[info]     at java.util.concurrent.FutureTask.run(FutureTask.java:166)
[info]     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
[info]     at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
[info]     at java.util.concurrent.FutureTask.run(FutureTask.java:166)
[info]     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
[info]     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
[info]     at java.lang.Thread.run(Thread.java:701)
```

```
[info]   - 09 when using 8 pairs with 10 messages *** FAILED *** (30 seconds, 6 milliseconds)
[info]     java.util.concurrent.TimeoutException: Futures timed out after [30 seconds]
[info]     at scala.concurrent.impl.Promise$DefaultPromise.ready(Promise.scala:219)
[info]     at scala.concurrent.impl.Promise$DefaultPromise.result(Promise.scala:223)
[info]     at scala.concurrent.Await$$anonfun$result$1.apply(package.scala:116)
[info]     at scala.concurrent.BlockContext$DefaultBlockContext$.blockOn(BlockContext.scala:53)
[info]     at scala.concurrent.Await$.result(package.scala:116)
[info]     at akka.typed.TypedSpec.await(TypedSpec.scala:40)
[info]     at akka.typed.TypedSpec.sync(TypedSpec.scala:58)
[info]     at akka.typed.PerformanceSpec$A$u0020static$u0020behavior$must$u0020be$u0020fast$.09$u0020when$u0020using$u00208$u0020pairs$u0020with$u002010$u0020messages(PerformanceSpec.scala:67)
[info]     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[info]     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
[info]     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[info]     at java.lang.reflect.Method.invoke(Method.java:622)
[info]     at org.scalatest.SpecLike$$anonfun$register$1$1$$anonfun$2.apply(SpecLike.scala:124)
[info]     at org.scalatest.SpecLike$$anonfun$register$1$1$$anonfun$2.apply(SpecLike.scala:122)
[info]     at org.scalatest.Transformer$$anonfun$apply$1.apply(Transformer.scala:22)
[info]     at org.scalatest.Transformer$$anonfun$apply$1.apply(Transformer.scala:22)
[info]     at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]     at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]     at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]     at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]     at org.scalatest.SpecLike$$anon$1.apply(SpecLike.scala:259)
[info]     at org.scalatest.Suite$class.withFixture(Suite.scala:1121)
[info]     at org.scalatest.Spec.withFixture(Spec.scala:1397)
[info]     at org.scalatest.SpecLike$class.invokeWithFixture$1(SpecLike.scala:256)
[info]     at org.scalatest.SpecLike$$anonfun$runTest$1.apply(SpecLike.scala:268)
[info]     at org.scalatest.SpecLike$$anonfun$runTest$1.apply(SpecLike.scala:268)
[info]     at org.scalatest.SuperEngine.runTestImpl(Engine.scala:306)
[info]     at org.scalatest.SpecLike$class.runTest(SpecLike.scala:268)
[info]     at org.scalatest.Spec.runTest(Spec.scala:1397)
[info]     at org.scalatest.SpecLike$$anonfun$runTests$1.apply(SpecLike.scala:314)
[info]     at org.scalatest.SpecLike$$anonfun$runTests$1.apply(SpecLike.scala:314)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:413)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]     at scala.collection.immutable.List.foreach(List.scala:381)
[info]     at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]     at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:390)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:427)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]     at scala.collection.immutable.List.foreach(List.scala:381)
[info]     at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]     at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:390)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:427)
[info]     at org.scalatest.SuperEngine$$anonfun$traverseSubNodes$1$1.apply(Engine.scala:401)
[info]     at scala.collection.immutable.List.foreach(List.scala:381)
[info]     at org.scalatest.SuperEngine.traverseSubNodes$1(Engine.scala:401)
[info]     at org.scalatest.SuperEngine.org$scalatest$SuperEngine$$runTestsInBranch(Engine.scala:396)
[info]     at org.scalatest.SuperEngine.runTestsImpl(Engine.scala:483)
[info]     at org.scalatest.SpecLike$class.runTests(SpecLike.scala:314)
[info]     at org.scalatest.Spec.runTests(Spec.scala:1397)
[info]     at org.scalatest.Suite$class.run(Suite.scala:1423)
[info]     at org.scalatest.Spec.org$scalatest$SpecLike$$super$run(Spec.scala:1397)
[info]     at org.scalatest.SpecLike$$anonfun$run$1.apply(SpecLike.scala:319)
[info]     at org.scalatest.SpecLike$$anonfun$run$1.apply(SpecLike.scala:319)
[info]     at org.scalatest.SuperEngine.runImpl(Engine.scala:545)
[info]     at org.scalatest.SpecLike$class.run(SpecLike.scala:319)
[info]     at akka.typed.TypedSpec.org$scalatest$BeforeAndAfterAll$$super$run(TypedSpec.scala:25)
[info]     at org.scalatest.BeforeAndAfterAll$class.liftedTree1$1(BeforeAndAfterAll.scala:257)
[info]     at org.scalatest.BeforeAndAfterAll$class.run(BeforeAndAfterAll.scala:256)
[info]     at akka.typed.TypedSpec.run(TypedSpec.scala:25)
[info]     at org.scalatest.tools.Framework.org$scalatest$tools$Framework$$runSuite(Framework.scala:444)
[info]     at org.scalatest.tools.Framework$ScalaTestTask.execute(Framework.scala:651)
[info]     at sbt.TestRunner.runTest$1(TestFramework.scala:76)
[info]     at sbt.TestRunner.run(TestFramework.scala:85)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1$$anonfun$apply$8.apply(TestFramework.scala:202)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1$$anonfun$apply$8.apply(TestFramework.scala:202)
[info]     at sbt.TestFramework$.sbt$TestFramework$$withContextLoader(TestFramework.scala:185)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1.apply(TestFramework.scala:202)
[info]     at sbt.TestFramework$$anon$2$$anonfun$$init$$1.apply(TestFramework.scala:202)
[info]     at sbt.TestFunction.apply(TestFramework.scala:207)
[info]     at sbt.Tests$.sbt$Tests$$processRunnable$1(Tests.scala:239)
[info]     at sbt.Tests$$anonfun$makeSerial$1.apply(Tests.scala:245)
[info]     at sbt.Tests$$anonfun$makeSerial$1.apply(Tests.scala:245)
[info]     at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
[info]     at sbt.std.Transform$$anon$3$$anonfun$apply$2.apply(System.scala:44)
[info]     at sbt.std.Transform$$anon$4.work(System.scala:63)
[info]     at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
[info]     at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:226)
[info]     at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
[info]     at sbt.Execute.work(Execute.scala:235)
[info]     at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
[info]     at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:226)
[info]     at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
[info]     at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
[info]     at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
[info]     at java.util.concurrent.FutureTask.run(FutureTask.java:166)
[info]     at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
[info]     at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
[info]     at java.util.concurrent.FutureTask.run(FutureTask.java:166)
[info]     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
[info]     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
[info]     at java.lang.Thread.run(Thread.java:701)
[WARN] [04/03/2015 04:40:14.187] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70726,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.188] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.188] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3613,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.188] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3613,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.188] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.188] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70725,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.189] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3612,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.189] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.189] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70726,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.189] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3612,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.189] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.189] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70724,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3612,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70724,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3612,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70724,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.190] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3612,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.191] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.191] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70725,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.191] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3613,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.191] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.191] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70724,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.191] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3613,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.192] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.192] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70725,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.192] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-2] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(3612,Actor[akka://PerformanceSpec/user/09/ponger-2#-230273215],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.192] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-1] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(50500,Actor[akka://PerformanceSpec/user/09/ponger-1#1539644193],Actor[akka://PerformanceSpec/user/09#87467718])
[WARN] [04/03/2015 04:40:14.192] [PerformanceSpec-akka.actor.default-dispatcher-2] [akka://PerformanceSpec/user/09/pinger-3] received dead letter from Actor[akka://PerformanceSpec/deadLetters]: Ping(70725,Actor[akka://PerformanceSpec/user/09/ponger-3#823438455],Actor[akka://PerformanceSpec/user/09#87467718])
[ERROR] [04/03/2015 04:40:14.192] [PerformanceSpec-akka.actor.default-dispatcher-23] [akka://PerformanceSpec/user/09] unexpected PostStop while waiting for 80 messages (got 50 valid ones)
akka.typed.StepWise$$anon$1: unexpected PostStop while waiting for 80 messages (got 50 valid ones)
	at java.lang.Throwable.<init>(Throwable.java:213)
	at java.lang.Exception.<init>(Exception.java:58)
	at java.lang.RuntimeException.<init>(RuntimeException.java:60)
	at java.lang.IllegalStateException.<init>(IllegalStateException.java:55)
	at akka.typed.StepWise$$anon$1.<init>(StepWise.scala:123)
	at akka.typed.StepWise$.akka$typed$StepWise$$throwIllegalState(StepWise.scala:123)
	at akka.typed.StepWise$$anonfun$akka$typed$StepWise$$behavior$1$1.applyOrElse(StepWise.scala:152)
	at akka.typed.StepWise$$anonfun$akka$typed$StepWise$$behavior$1$1.applyOrElse(StepWise.scala:145)
	at akka.typed.ScalaDSL$Full.management(ScalaDSL.scala:146)
	at akka.typed.ActorAdapter.postStop(Impl.scala:56)
	at akka.actor.Actor$class.aroundPostStop(Actor.scala:475)
	at akka.typed.ActorAdapter.aroundPostStop(Impl.scala:16)
	at akka.actor.dungeon.FaultHandling$class.akka$actor$dungeon$FaultHandling$$finishTerminate(FaultHandling.scala:210)
	at akka.actor.dungeon.FaultHandling$class.handleChildTerminated(FaultHandling.scala:292)
	at akka.actor.ActorCell.handleChildTerminated(ActorCell.scala:369)
	at akka.actor.dungeon.DeathWatch$class.watchedActorTerminated(DeathWatch.scala:63)
	at akka.actor.ActorCell.watchedActorTerminated(ActorCell.scala:369)
	at akka.actor.ActorCell.invokeAll$1(ActorCell.scala:455)
	at akka.actor.ActorCell.systemInvoke(ActorCell.scala:478)
	at akka.dispatch.Mailbox.processAllSystemMessages(Mailbox.scala:282)
	at akka.dispatch.Mailbox.run(Mailbox.scala:223)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)

```

--- a/akka-typed/src/test/scala/akka/typed/TypedSpec.scala
+++ b/akka-typed/src/test/scala/akka/typed/TypedSpec.scala
@@ -37,7 +37,8 @@ class TypedSpec(config: Config) extends Spec with Matchers with BeforeAndAfterAl
   }
 
   // TODO remove after basing on ScalaTest 3 with async support
-  def await[T](f: Future[T]): T = Await.result(f, 30.seconds)
+  import akka.testkit._
+  def await[T](f: Future[T]): T = Await.result(f, 60.seconds.dilated(system.untyped))
 
   val blackhole = await(system ? Create(Props(ScalaDSL.Full[Any] { case _ ⇒ ScalaDSL.Same }), "blackhole"))
 

